alignak_backend_client/client.py

```diff
@@ -56,7 +55,7 @@
 else:
     logging.basicConfig()
 # Set logger level to WARNING, this to allow global application DEBUG logs without being spammed...
-logger.setLevel(WARNING)
+logger.setLevel('DEBUG')
```

Comment:
Sure about the log level ? Then update the comment according...

Reply:
Just for tests, I'll change it in next commit